src/js/components/Tab/index.d.ts

```diff
@@ -2,6 +2,7 @@ import * as React from "react";
 import { Omit } from "../../utils";
 
 export interface TabProps {
+  disabled?: boolean;
```

Comment:
If you are using JSX.IntrinsicElements['button'] then the type of `disabled` is derived from the HTML element, so I'm not sure if this is needed.